src/open_inwoner/templates/pages/profile/me.html

```diff
@@ -186,41 +184,38 @@ <h4 class="card__heading-4"><span class="link link__text">{% trans "Mijn Interes
                     <a href="#" class="card card--compact card--stretch" id="profile-section-mentors">
                 {% endif %}
                         <div class="card__body">
-                            <h4 class="card__heading-4"><span class="link link__text">{% trans "Mijn begeleider" %}</span></h4>
-                            {% render_list %}
-                                <span class="profile-list ellipsis">
+                            <div class="ellipsis">
+                                <h4 class="card__heading-4"><span class="link link__text">{% trans "Mijn begeleider" %}</span></h4>
+                                {% render_list %}
                                     {% for name in mentor_contact_names %}
                                         {% list_item text=name compact=True strong=False %}
                                     {% empty %}
-                                        <p class="p list_paragraph">{% trans "U heeft geen gemeentelijke begeleider." %}</p>
+                                        <li class="card__text--small">{% trans "U heeft geen gemeentelijke begeleider." %}</li>
                                     {% endfor %}
-                                </span>
-                            {% endrender_list %}
+                                {% endrender_list %}
 
-                            {% if inbox_page_is_published %}
-                                <span class="link link--icon link--secondary" aria-label="{% trans "Stuur een bericht" %}" title="{% trans "Stuur een bericht" %}">
+                                {% if inbox_page_is_published %}
+                                    <span class="link link--icon link--secondary" aria-label="{% trans "Stuur een bericht" %}" title="{% trans "Stuur een bericht" %}">
```

Comment:
Do we need the `aria-label` here, since we have "Stuur een bericht" immediately below?

Reply:
That is actually an issue that will come up with the other thirty accessibility tasks still to come, but I will correct it on the profiel page here anyway.
Important to note: aria-label does not work on static elements such as div or span; it is meant for interactive elements that need a label, like buttons.
(The OIP project is quite cluttered with them...).